lib/blocs/messages_cubit/messages_cubit.dart

```diff
@@ -69,6 +69,20 @@ abstract class BaseMessagesCubit extends Cubit<MessagesState> {
     }
   }
 
+  Future<void> swipeReply(
+    String threadId,
+  ) async {
+    Message parentMessage;
+
+    parentMessage = await _repository.getMessage(messageId: threadId);
```

Comment:
```
final parentMessage = await _repository.getMessage(messageId: threadId);
```

Reply:
I think that would be more professional :)

Reply:
Yes, I will remove -  Message parentMessage;